core/modules/node/js/node.preview.js

```diff
@@ -0,0 +1,28 @@
+
+(function ($) {
+
+Backdrop.behaviors.nodePreview = {
+  attach: function (context) {
+    // Insert the preview admin banner into the page before the layout.
+    $('.node-preview-container').each(function() {
+      $(this).insertBefore($('.page').once('preview-js').find('.layout'));
```

Comment:
Instead of tying to the "layout" class, we could take the second-highest parent and use that. e.g.

`$topParent = $(this).parents(':not(body)').last();`

This is also a slightly unusual use of `.once()`, just to add a class. If we could do the `.once()` on `.node-preview-container`, that would be more inline with most other behavior attachments. e.g.

```
 $('.node-preview-container').once('node-preview').each(function() {
  $topParent = $(this).parents(':not(body)').last();
  $topParent.addClass('preview-js');
  $topParent.on('click.preview', 'a', function (e) {
    ...
  }
});
```

Reply:
done